tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -31,6 +31,7 @@
 class TestAzureRMComputeResourceTestCase:
     """Tests for ``api/v2/compute_resources``"""
 
+    @pytest.mark.e2e
```

Comment:
A CRUD (Create, Read, Update, Delete) test by itself is not an end-to-end test, as it typically only focuses on testing the individual functionalities of creating, reading, updating, and deleting data within the application. However, a CRUD test could be a part of an end-to-end test, which would involve testing the entire workflow of a user interacting with the application and performing the CRUD operations as part of that workflow.

e.g. `Compute Resource` Create, Update, and Delete do not make any value for the customer as end to end. They are not actually using the any feature. But compute resource used for host provisioning make as end to end test. Think like a customer here, will any cusotmer will just create Compute Resource and update and delete it? Customer will use compute resource with host provisoning then it make sense as end to end test. 

Please let me know yout thoughts here ?

Reply:
Agree. Updated the test markers for the compute resources components.

Reply:
We can remove this as e2e.